asv/util.py

```diff
@@ -358,33 +370,40 @@ def sig_forward(signum, frame):
             signal.signal(signal.SIGTSTP, sig_forward)
             signal.signal(signal.SIGCONT, sig_forward)
 
-        fds = {
-            proc.stdout.fileno(): stdout_chunks,
-            proc.stderr.fileno(): stderr_chunks
-            }
-
-        while proc.poll() is None:
+            fds = {
```

Comment:
There are systems that don't have setpgid that are still Unix-like and can still use select.  It looks like you've changed the behavior here so anything that doesn't have setpgid doesn't use select (and thus can't provide feedback that the subprocess is running).  Can you fix the logic here so we only fall back to `proc.communicate` on Windows?